e2e-tests/tests/tablespaces/08-assert.yaml

```diff
@@ -6,7 +6,7 @@ apiVersion: postgres-operator.crunchydata.com/v1beta1
 kind: PostgresCluster
 metadata:
   name: tablespaces
-  generation: 3
+  generation: 4
```

Comment:
But, why do we have it? @pooknull @inelpandzic ^

Reply:
In this PR, when creating a manual backup we add a new flag to the `.spec.backups.pgbackrest.manual.options` field: `--annotation="percona.com/backup-name"="tablespaces-full"`

In our `cr.yaml` this field has only `--type=full`: https://github.com/percona/percona-postgresql-operator/blob/5400ec12e78d7fbace88e2859d2ba4b98041c77d/deploy/cr.yaml#L322-L325

When we create a cluster with our default `cr.yaml`, the `postgrecluster` resource will have the following fields:
```
spec:
  backups:
    pgbackrest:
      manual:
        options:
        - --type=full
        repoName: repo1
```

After creating a backup, it will change to:
```
spec:
  backups:
    pgbackrest:
      manual:
        options:
        - --type=full
        - --annotation="percona.com/backup-name"="tablespaces-full"
        repoName: repo1
```

This is causing an additional generation